redbot/cogs/general/general.py

```diff
@@ -195,7 +195,7 @@ def __init__(self):
             [
                 m.status
                 for m in guild.members
-                if m.status == discord.Status.online or m.status == discord.Status.idle
+                if m.status != discord.Status.offline
             ]
```

Comment:
I think the formatter will be happy if you put the list comprehension all on one line